iib/workers/tasks/opm_operations.py

```diff
@@ -500,6 +500,38 @@ def opm_registry_deprecatetruncate(base_dir: str, index_db: str, bundles: List[s
     run_cmd(cmd, {'cwd': base_dir}, exc_msg=f'Failed to deprecate the bundles on {index_db}')
 
 
+def deprecate_bundles_fbc_containerized(
+    bundles: List[str],
+    base_dir: str,
+    binary_image: str,
+    index_db_file: str,
+) -> None:
+    """
+    Deprecate the specified bundles from the FBC index image.
+
+    Dockerfile is created only, no build is performed.
+
+    :param list bundles: pull specifications of bundles to deprecate.
+    :param str base_dir: base directory where operation files will be located.
+    :param str binary_image: binary image to be used by the new index image.
+    :param str index_db_file: path to SQLite index.db which should be used for deprecatetruncate.
+    """
+    conf = get_worker_config()
+
+    # Break the bundles into chunks of at max iib_deprecate_bundles_limit bundles
+    for i in range(
```

Comment:
This is identical to line 555 - 566 in opm_operations.py in `deprecate_bundles_fbc`, consider replacing that part with a call to this function? 

Also consider renaming one of the functions? If you do not want to rename them (as the old code won't be used for much), add a brief comment in deprecate_bundles_fbc